examples/c_hello/main.c

```diff
@@ -17,5 +17,5 @@ static void nop(
 
 int main(void) {
   putnstr_async(hello, strlen(hello), nop, NULL);
-  return 0;
+  yield();
```

Comment:
```suggestion
  // Because we used the async method (as opposed to something synchronous,
  // such as printf), we must explicitly wait for the asynchronous write to complete. 
  yield();
  // Now we are done.
  return 0;
```

I think this is a bit of a weird example, and being that it's the `c_hello` example---i.e., one of the simplest---some extra explanation is warranted here.